target/linux/ramips/mt7620/base-files/etc/board.d/02_network

```diff
@@ -175,7 +175,7 @@ ramips_setup_interfaces()
 		;;
 	lava,lr-25g001)
 		ucidef_add_switch "switch0" \
-			"1:lan" "2:lan" "3:lan" "4:lan" "5:wan" "0@eth0"
+			"1:lan:4" "2:lan:3" "3:lan:2" "4:lan:1" "5:wan" "0@eth0"
```

Comment:
this change makes it the same as `engenius,esr600` at line 140

should remove this block and add the lava board to that instead

Reply:
Done.